Model/ProductProtection.php

```diff
@@ -408,9 +408,12 @@ private function upsert(
             }
 
             $options = $this->createOptions($product, $item, $optionValues);
-            $item->setOptions($options);
-
-            $this->addAdditionalOptions($item, $productId, $term, $leadToken, $leadQuantity);
+            if (isset($options)) {
```

Comment:
Why would `$options` not be set? Does `isset()` check for empty as well?

Reply:
Removed this check